data/transactions/logic/assembler.go

```diff
@@ -2565,15 +2567,15 @@ func parseSwitch(program []byte, pos int) (targets []int, nextpc int, err error)
 		return
 	}
 	pos += bytesUsed
-	if numOffsets > uint64(len(program)) {
-		err = errTooManyItems
+	if numOffsets > math.MaxUint16 {
```

Comment:
@jannotti Based on https://github.com/algoidurovic/go-algorand/pull/3/commits/c066ff29baec7d5347891e7264388111d15fbd80, I'd expect a limit change here too.  Can you double check me?

* If you take the suggestion:  Note, there's also an import change (remove `math`).  I attached a patch for convenience:  [diff.txt](https://github.com/algoidurovic/go-algorand/files/9578323/diff.txt)
* If you agree with the suggestion, I'd consider applying a constant though I don't feel strongly.


```suggestion
	if numOffsets > 256 {
```

Reply:
https://github.com/algoidurovic/go-algorand/pull/3/commits/d705b0c1d12a53587190a1e763d064d8f647138e resolves the thread for me.  Keep in mind, I don't have resolve access.